{% load staticfiles %}
<html>

<head>
  <title>RPGeNet v2.0 - Retinitis Pigmentosa Gene Network</title>
  <meta name="description" content="RPGeNet v2.0 - Retinitis Pigmentosa Gene Network: interacting with the network of genes related to retinitis pigmentosa genes.">
  <meta name="keywords" content="RPGeNet,retinitis pigmentosa,RP,interaction network,retinal disease,driver genes">
  <link rel="shortcut icon" href="{%  static 'Images/favicon.png' %}">
  <link rel="stylesheet" href="{% static 'css/style.css' %}">
  <link rel="stylesheet" href="{% static 'css/bootstrap-min.css' %}">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
</head>


<body>

  <div id="title-div">
    <div id="title-container" class="row">
      <div class="col-sm-2" id="app-logo-div">
        <a href="/RPGeNet"><img id="app-logo" src="{% static 'Images/logo.png' %}"></a>
      </div>
      <div class="col-sm-10" id="title-text"> RPGeNet <span class="version">v2.0</span> </div>
    </div>


  </div>

  <div class="navbar-container">
    <nav class="navbar navbar-default thenavbar">
      <div class="navbar-inner">
        <ul class="nav navbar-nav">
          <li id="navbar-home" class="nav-elem"><a href="{% url index_view %}">Home</a></li>
          <li id="navbar-data" class="nav-elem"><a href="{% url data %}">Data</a></li>
          <li id="navbar-tutorial" class="nav-elem active"><a href="{% url tutorial %}">Tutorial</a></li>
          <li id="navbar-about" class="nav-elem"><a href="{% url about %}">About</a></li>
        </ul>
      </div>
    </nav>
  </div>
  <br>

  <div class="body-card">
    <h1>Tutorial</h1>
    <hr>

    <div class="toc">
      <h3>Table of Contents</h3>

      <ol>
        <li><a href="#network-explorer">Network Explorer</a></li>
          <ul>
            <li><a href="#network-explorer-form">Form Options</a></li>
          <ol>
              <li><a href="#form-gene">Gene/s</a></li>
              <li><a href="#form-int">Interaction Level</a></li>
              <li><a href="#form-dist">Gene distance</a></li>
              <li><a href="#form-exp">Expression</a></li>
            </ol>
          <li><a href="#controls">Navigation options</a></li>
            <ol>
              <li><a href="#options-home">Home</a></li>
              <li><a href="#options-click">On Click</a></li>
              <li><a href="#options-layout">Layout Selection</a></li>
              <li><a href="#options-connect">Connect Genes</a></li>
              <li><a href="#options-border">Border size</a></li>
              <li><a href="#options-export">More Controls</a></li>
              <li><a href="#options-legend">Legend and Tutorial</a></li>
            </ol>
          <li><a href="#legend">Legend</a></li>
        </ul>
        <li><a href="#pathway-to-level">Pathway to Level</a></li>
        <li><a href="#shortest-path">Shortest Path Between Genes</a></li>
        <li><a href="#guided-tutorial">Guided Tutorial with CERKL</a></li>
          <ul>
            <li><a href="#network-explorer-guided">Network Explorer</a></li>
            <li><a href="#pathway-guided">Pathway to Level</a></li>
            <li><a href="#shortest-guided">Shortest Path Between Genes</a></li>
          </ul>
      </ol>
    </div>




  <h3 id="network-explorer">Network Explorer</h3>
  <hr>

   <p>
    The Network Explorer allows you to explore the different levels of the Retinitis Pigmentosa (RP) network by selecting one or several genes and retrieving all their interactions.
   </p>

  <br>
  <h4 id="network-explorer-form">Form options</h4>

    <div class="center-text">
      <a href="{% static 'Images/network_explorer_form.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/network_explorer_form.png' %}">
      </a>
    </div>

   <br><br>
   <ol>
    <li id="form-gene"><b>Gene/s</b>. </li>
    <p>
      Here is where you can input one or several genes of interest related or unrelated to Retinitis Pigmentosa (RP) (althought if is not related there is a big change that it won't have interactors, since the network is using RP genes as seeds). If the input are multiple genes, then those genes have to be separated be a comma "," or a blank space " " between them.
    </p>
    <li id="form-int"><b>Interaction Level</b>. </li>
    <p>
      The RP network has 4 levels: "Skeleton", Level 1, Level 2, Level 3, and WholeGraph. Each level represents a sub-network of genes and interactions ranging from more related to Retinitis Pigmentosa (Skeleton) to less related (WholeGraph).
    </p>
    <li id="form-dist"><b>Gene distance</b>. </li>
    <p>
      Specify the distance from the input genes to retrieve from the Network. By default this is set to 1, that is, RPGeNet will only retrieve parents and childs of the specified genes. If set to 2, RPGeNet will also retrieve parents and childs to the interactors of the input genes. Please, note that this field can only be changed when Interaction Level is set to Skeleton.
    </p>
    <li id="form-exp"><b>Expression</b>. </li>
    <p>
      Color the genes in the graph according to their expression in the specified experiment.
    </p>

   </ol>

  <br><br>
  <h4 id="controls">Navigation Options</h4>
  <div class="center-text">
    <a href="{% static 'Images/guided_tutorial/navoptions2.png' %}" target="_blank">
      <img class="tutorial-img" src="{% static 'Images/guided_tutorial/navoptions2.png' %}">
    </a>
  </div>
   In Network Explorer you have several options and controls to modify the graph visualization.

   <br><br>
   <ol>
     <li id="options-home"><b>Home:</b></li>
     <p>
        The logo image is a button to return to the home page.
      </p>
     <li id="options-click"><b>On Click:</b></li>
     <p>
        This option defines three different behaviours when clicking on Nodes or Edges. "Show Properties" will display an overlay with information about the gene or the interaction. "Node Addition" will add all the interactors to the clicked node to the visualization. Finally, "Node Deletion" will delete the clicked node from the visualization; please, not that this process is irreversible.
      </p>
     <li id="options-layout"><b>Layout Selection:</b></li>
     <p>
      Change the layout of the nodes and interactions. Available layouts are: 'Dagre', 'Cose', 'Grid', 'Circle', and 'Concentric'.
     </p>
     <li id="options-connect"><b>Connect Genes:</b></li>
     <p>
        This button will add all the interactions in the current level between all the genes displayed on the visualization.
      </p>
     <li id="options-unconnect"><b>Remove Unconnected Genes:</b></li>
     <p>
        This button will remove all the nodes that are currently not connected in the graph display.
      </p>
     <li id="options-border"><b>Border size:</b></li>
     <p>
      Increase or decrease the border size of genes.
     </p>
     <li id="options-node-size"><b>Node size:</b></li>
     <p>
      Increase or decrease the border size of genes.
     </p>
     <li id="options-export"><b>More Controls:</b></li>
     <ul>
        <li><b>Fit:</b> Fit all the nodes in the drawing area to make them visible.</li>
        <li><b>Image:</b> Export a png image of the graph.</li>
        <li><b>Export:</b> Export the graph as a csv file to be used elsewhere (e.g.: Excel).</li>
        <li><b>Save:</b> Save the graph in JSON format to be able to upload it later.</li>
        <li><b>Upload:</b> Upload a previously exported JSON graph (using the 'Export' button).</li>
     </ul>
     <li id="options-legend"><b>Legend and Tutorial:</b></li>
      <ul>
        <li><b>Legend:</b> This button opens a window with a legend detailing the meaning of the shapes and colors within the network explorer.</li>
        <li><b>Tutorial:</b> This buttons sends you to the tutorial page you are currently on</li>
     </ul>
   </ol>

   <br>
   <h4 id="legend">Legend</h4>
   <p>
     The roots of the RPGeNet network begin with the driver genes of retinitis
     pigmentosa. All driver genes are purple borders. Some driver genes are
     associated with syndromic genetic disorders and are square shaped within the
     network explorer. Non-syndromic driver genes are diamond shaped. Driver genes
     that are syndromic and non-syndromic are star shaped.
   </p>
   <div class="center-text">
     <a href="{% static 'Images/guided_tutorial/levels.png' %}" target="_blank">
       <img class="rpgenet_legend" src="{% static 'Images/guided_tutorial/driver_nodes_nb.png' %}">
     </a>
   </div>
   <br>
   <p>
     Non-driver nodes are all circular and have borders that indicate the level, where
     the gene is found within the network, by the color.
   </p>
   <div class="center-text">
     <a href="{% static 'Images/guided_tutorial/levels.png' %}" target="_blank">
       <img class="rpgenet_legend" src="{% static 'Images/guided_tutorial/levels_nb.png' %}">
     </a>
   </div>
   <p>
     The arrows that connect nodes within the network are color-coded to indicate
     whether they are physical, genetic or unknown interactions.
   </p>
   <br>
   <div class="center-text">
     <a href="{% static 'Images/guided_tutorial/interactions.png' %}" target="_blank">
       <img class="rpgenet_legend" src="{% static 'Images/guided_tutorial/interactions_nb.png' %}">
     </a>
   </div>
   <br>
   <p>
     All nodes, driver and non-driver genes, have a colored interior that represents
     their expression depending on the expression data used. Absolute expression is
     the raw expression within the retina. The RET-ALL option is the relative expression
     within the retina in comparison with all 32 tissues of the microarray experiment.
     The RET-OTHER is the relative expression within the retina in comparison with all
     the tissues minus the retina.
   </p>
   <div class="center-text">
     <a href="{% static 'Images/expression-legend.png' %}" target="_blank">
       <img class="rpgenet_legend" src="{% static 'Images/expression-legend.png' %}">
     </a>
   </div>

  <br><br>
  <h3 id="pathway-to-level">Pathway to Level</h3>
  <hr>
  <div class="center-text">
    <a href="{% static 'Images/pathway_form.png' %}" target="_blank">
      <img class="tutorial-img-large" src="{% static 'Images/pathway_form.png' %}">
    </a>
  </div>
  <br>
  <p>
    Pathway to Level allows you to get the shortest paths between a specified Gene and either 'Driver Genes' or 'Skeleton Genes'. If there is more than one target gene with the same path length (the same number of interactions until reaching that gene), all the pathways will be displayed on the results page.
  </p>
  <p>
    Pathway to level is useful when trying to assess the importance of a particular gene to Retinitis Pigmentosa, or to have an idea of how close a gene is to other genes related to Retinitis Pigmentosa (in terms of signalling and protein pathways).
  </p>

  <div class="center-text">
    <a href="{% static 'Images/pathway_results.png' %}" target="_blank">
      <img class="tutorial-img-large" src="{% static 'Images/pathway_results.png' %}">
    </a>
  </div>
  <br>
  <p>
    As can be seen in the image above, each pathway is represented as an individual graph visualization. Only one pathway is represented between two genes; if you want all the possible shortest pathways between, for example, PMM2 and SNRNP200, you can click on the "All shortest paths" link above the visualization. In addition, if you want to explore a particular pathway further, you can click on the "Explore Network" button, which will bring you to Network Explorer with the specified pathway in the graph visualization.
  </p>


  <br><br>
  <h3 id="shortest-path">Shortest Path Between Genes</h3>
  <hr>
  <div class="center-text">
    <a href="{% static 'Images/shortest_form.png' %}" target="_blank">
      <img class="tutorial-img-large" src="{% static 'Images/shortest_form.png' %}">
    </a>
  </div>

  <br>
  <p>
    This form allows you to compute the shortest path between any two genes in the network. Please, note that all the connections in RPGeNet are directional, so the results of A->B will be different from B->A. All the shortest paths are computed on the WholeGraph. The results page from this form is the same as the one from "Pathway to Level".
  </p>


  <h3 id="guided-tutorial">Guided Tutorial with CERKL</h3>
  <hr>
    <p>
    This guided tutorial will show you how to:
      <ol>
        <li>find your gene of interest in the network</li>
        <li>learn how to edit, expand or contract a graph in network explorer</li>
        <li>learn to set and change the expression visualized</li>
        <li>learn to find the shortest paths between two genes of interest</li>
        <li>learn to find all the pathways between your genes of interest to drivers/skeleton</li>
      </ol>
    </p>
    <p>
      Starting from the RPGeNet homepage we see three options available to users
      so you can use: Network Explorer, Pathway to Level and Shortest Paths. Below
      we will go through each on of these options and guide you through the process.
    </p>

    <h4 id="network-explorer-guided">Network Explorer</h4>
    <p>
      The first option available is the Network Explorer. This option is great
      for when you have a gene of interest and want to know what reacts with your
      gene, when you want to create a graph of the interactions with your gene
      and when you want to explore network with your gene of interest as the focal
      starting point. If we take a look at Network Explorer we are asked to type
      in our gene(s) of interest, choose the interaction level,
      the gene distance and the expression dataset.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/netexplorer.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/netexplorer.png' %}">
      </a>
    </div>
    <p>
      In the gene section, we will type in the gene CERKL but do note that you
      can type in multiple genes and not only one.
    </p>
    <p>
      The interaction level is basically asking us to chose a level of penetration
      into the network you wish to search through and how complex of a graph you
      want returned. The most basic level in the options is the skeleton. The
      skeleton is composed of all the shortest pathways between the driver genes
      of retinitis pigmentosa and are the most relevant interactions related to
      retinitis pigmentosa. If we pick the skeleton, Network Explorer will only
      search through the interactions of the skeleton graph and the output
      will only include interactions, of your gene of interest, found in the skeleton.
      Each level above the skeleton adds the parent and children nodes of the previous level
      that do not exist within the previous level(s) of the network. Depending on
      the level you choose, you will have more genes, interactions and pathways to
      search through but also an increase in noise as you move further away to
      interactions not as vital to retinitis pigmentosa.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/Menu_007.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/Menu_007.png' %}">
      </a>
    </div>
    <p>
      For the purpose of this guided tutorial, we will keep it simple and chose skeleton.
    </p>
    <p>
      The gene distance option is only available if the skeleton is chosen as the
      interaction level. This option is simply there to save time and automatically
      add the parents and children of all the genes that interact with our gene of
      interest in the skeleton. Note that only parents/children found within the
      skeleton will be added if there are any. We will keep the defaul of one.
    </p>
    <!--<img class="" src="{% static 'Images/' %}">-->
    <p>
      Finally, we must choose the expression data. There are four options,all from
      the same microarray dataset, GSE7905, that have been uniquely calculated.
      The first option is the absolute expression values of genes within the retina
      only. The second option is the relative expression between retina and
      all the 32 tissues analyzed. The third option is the relative expression
      values between retina and all non-retinal tissues. Lastly, the final option
      is the relative expression between retina and the liver.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/Menu_009.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/Menu_009.png' %}">
      </a>
    </div>
    <p>
      We will chose the absolute expression option and then click on explore network.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/retabsdisplay2.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/retabsdisplay2.png' %}">
      </a>
    </div>
    <p>
      The image above is the result of our input: a graph with our gene of interest
      as the focal point and all genes that interact with it in the skeleton. You
      can also see there is a control panel to the righ hand side and other tools
      you can use to help you navigate the network and find what you are looking
      for.
    </p>
    <p>
      Lets take a look at the control panel to the right hand side of the screen.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/command2.png' %}" target="_blank">
        <img class="rpgenet_legend" src="{% static 'Images/guided_tutorial/command2.png' %}">
      </a>
    </div>
    <p>
      The control panel offers helpful tools to help navigate the network, edit
      your graph and save the graph as an image or for later use. We will go through
      each button starting from the top. The first section, called "On click",
      has three buttons and affects the output of your click when clicking on
      nodes of your graph.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/on-click.png' %}" target="_blank">
        <img class="tutorial-img" src="{% static 'Images/guided_tutorial/on-click.png' %}">
      </a>
    </div>
    <p>
      The first button, "Show Properties", is the default setting and is used to
      find more information about a gene or an interaction. When on "Show Properties",
      all you need to do is click a gene that interest you and a popup window will
      open.You will find helpful information like its alias, a summary
      about the gene, gene data, expression data, its gene ontology and links to
      other well known gene databases for more information. The image below is
      the result of clicking on the gene VHL, when using "Show Properties".
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/show-properties-result.png' %}" target="_blank">
        <img class="tutorial-img" src="{% static 'Images/guided_tutorial/show-properties-result.png' %}">
      </a>
    </div>
    <p>
      We can play with our graph and add or delete nodes. If we click on the node addition
      button and then on a gene that interests us, we can add the parents and child
      of that gene. Go ahead and click on "Node Addition" and then click the genes
      PPM1A and PPM1B.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/node-addition-result2.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/node-addition-result2.png' %}">
      </a>
    </div>
    <p>
      If there is a gene that does not interest you, you can click on node deletion
      and double click on that gene to delete it. If there are many genes you want
      to delete, you can press and hold shift and draw a highlighting box around
      those genes and then click on one of the genes highlighted to delete them all.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/node-deletion-howtodrag2.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/node-deletion-howtodrag2.png' %}">
      </a>
    </div>
    <p>
      There is also an option to change the layout of the graph. There are ten
      different layout options you can pick from. Go ahead and pick one by one
      to see which one you like the most
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/layout.png' %}" target="_blank">
        <img class="tutorial-img" src="{% static 'Images/guided_tutorial/layout.png' %}">
      </a>
    </div>
    <p>
      When we add new nodes to the graph they only show their interaction with the
      gene you clicked. If you want to see how the added node connects with the
      rest of the graph, you can click "Connect Genes". You can even click the same
      button before adding nodes to see if the genes that interact directly with
      you gene of interest interact with each other.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/connect-genes-result2.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/connect-genes-result2.png' %}">
      </a>
    </div>
    <p>
      When you delete nodes, you may end up with many unconnected nodes in the
      graph display. We can use the Remove Unconnected button to remove any unconnected
      nodes. Restarting with the original CERKL input on the netexplorer, we are
      going to click on "Node Addition" and click on the gene SIRT1. We will then
      click on "Node Deletion" and delete SIRT1. All of nodes connecting to SIRT1
      will be unconnected. Now click "Remove Unconnected" and all the unconnected
      nodes should be deleted from the graph display.
    </p>
    <a href="{% static 'Images/guided_tutorial/REMOVE_UNCONNECTED_ANIMATION.gif' %}" target="_blank">
        <img class="tutorial-gif-large" src="{% static 'Images/guided_tutorial/REMOVE_UNCONNECTED_ANIMATION.gif' %}">
      </a>
    <p>
      As you can see, each node has a border that tells us the lowest level the
      gene is found. The size of the border can be modified using "Border Size".
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/border-size.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/border-size.png' %}">
      </a>
    </div>
    <p>
      You may have noticed that the nodes have different sizes. The size corresponds
      to the number of known variants. If you would like to turn off the node size
      you can click "Disable" under "Node Size" on the right hand ....
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/node-size.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/node-size.png' %}">
      </a>
    </div>
    <p>
      Under "Controls" there are many useful features you can use. The "Fit" button
      resizes your graph so that it fits on screen. If you want an image of your graph, you can click on the "Image" button and
      a PNG image of your graph will be created for you to download. If you want to a list of the interactions in your graph, you can click the
      "Export" button that and a TBL file will be created and can then be opened
      on Excel.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/controls.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/controls.png' %}">
      </a>
    </div>
    <p>
      If you would like to save your current graph for later use, you can click
      "Save" and a JSON file will be created for you to download. Give it a try
      and click "Save".This JSON file can then be uploaded to RPGeNet. If we
      restart and go back to the Homepage, we can click on "Explore Network"
      without typing any gene.You will get a "No Results" warning but can click
      on "Upload Graph" to upload the JSON file.
    </p>
    <!--<div class="center-text">
      <a href="{% static 'Images/guided_tutorial/controls.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/controls.png' %}">
      </a>
    </div>-->
    <p>
      You can also retype CERKL and get the standard graph and then upload using
      the "Upload" button under "Controls".
    </p>
    <p>
    If you need help or are lost, the legend and tutorial buttons are available,
    found at the bottom of the control panel.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/legend-tutorial.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/legend-tutorial.png' %}">
      </a>
    </div>
    <p>
    You will notice that the nodes have different shapes and colors as do the
    interactions. We can quickly check what the colors and shapes mean by looking
    at the legend, that is found scrolling down the control panel on the right.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/rpgenet_legend.png' %}" target="_blank">
        <img class="rpgenet_legend" src="{% static 'Images/rpgenet_legend.png' %}">
      </a>
    </div>
    <p>
      If you decide, whiles exploring the network or fixing your graph, that you
      want to see different expression values, you can always click under "Expression",
      on the top right hand side. All four options are available to chose from.
      Currently we have "ABSOLUTE" expression in the retina. Why don't we change
      it to relative expression "RET-ALL".
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/expression.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/expression.png' %}">
      </a>
    </div>
    <p>
      You can see that the color on the inside of the nodes changes. The absolute
      option has a different coloring scheme than the relative options. You can
      click on "Show Legend" for the color scale.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/retalldisplay2.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/retalldisplay2.png' %}">
      </a>
    </div>

    <h4 id="pathway-guided">Pathway to Level</h4>
    <p>
      The second option available is Pathway to Level. This option is used when
      you want to find all the pathways that connect your gene of interest to the
      driver genes of retinitis pigmentosa or to the skeleton level of the network.
      This is useful for genes that may not be a driver or part of the skeleton
      and you want to predict how relevant it may be.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/getpath.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/getpath.png' %}">
      </a>
    </div>
    <p>
      You can type in a gene of interest or try out the gene "RAX". In the "Path to"
      section, you must chose between finding a pathway to the skeleton or to driver
      genes.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/pathto.png' %}" target="_blank">
        <img src="{% static 'Images/guided_tutorial/pathto.png' %}">
      </a>
    </div>
    <p>
      We will keep the default, drivers, for the "Path to" section. Like in Network
      Explorer, you also have to chose the expression data. We will also keep the
      default, ABSOLUTE, and click "Get Path to Level".
    </p>
    <!--<img class="legend-netexp" src="{% static 'Images/' %}">-->
    <p>
      The output should be a list of pathways between the gene inserted and drivers
      if the pathways exist. In the case of RAX, we have three pathways.
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/getpathwayoutput.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/getpathwayoutput.png' %}">
      </a>
    </div>
    <p>
      When you find a pathway that interests you, you can transfer the pathway to
      Network Explorer by clicking "Explore Network" on the top right of the
      pathway. The output should be your chosen pathway on Network Explorer. Here you can
      use all the tools we used in the Network Explorer tutorial (above).
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/getexplr-rax2.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/getexplr-rax2.png' %}">
      </a>
    </div>
    <h4 id="shortest-guided">Shortest Path Between Genes</h4>
    <p>
      The final option is the Shortest Path Between Genes. This option is great
      for finding all the pathways between two genes of interest. This time you
      have two gene inputs: "Source gene" and "Target gene". Lets use CERKL as our
      source gene and RHO as our target gene. Do note that Shortest Paths is
      directional and so you need to type the genes in order from upstream (source)
      to downstream (target). If you want to see the opposite order, you must
      restart the query and switch the order of the genes. Like in the previous
      tools, you were asked to pick the expression data. You have the same four
      options and can choose the one best for you or keep the default for this
      tutorial. Than click "Get Shortest Paths".
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/shortestpath.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/shortestpath.png' %}">
      </a>
    </div>
    <p>
      Like in Pathway to Level, we get a list of pathways, if they exist. In the
      CERKL to RHO example, there are three shortest pathways. We can transfer
      the pathway that interest us to Network Explorer by clicking "Exlore Network".
    </p>
    <div class="center-text">
      <a href="{% static 'Images/guided_tutorial/netexplr-cerkl-rho2.png' %}" target="_blank">
        <img class="tutorial-img-large" src="{% static 'Images/guided_tutorial/netexplr-cerkl-rho2.png' %}">
      </a>
    </div>
    <p></p>
    <br>
    <p>
      This concludes our guided tutorial of RPGeNet.
    </p>

  </div>






  <div id="footer">
    <div class="footer-left">
      <div class="cgl-logo-footer-container">
        <a target="_blank" href="https://compgen.bio.ub.edu">
          <img class="cgl-logo-footer" src="{% static 'Images/compgen-logo.png' %}">
        </a>
      </div>
      <div class="cgl-links-footer">
        <a target="_blank" href="https://compgen.bio.ub.edu">Home</a> ·
        <a target="_blank" href="https://compgen.bio.ub.edu/Research">Research</a> ·
        <a target="_blank" href="https://compgen.bio.ub.edu/Acknowledgements">Acknowledgements</a> ·
        <a target="_blank" href="https://compgen.bio.ub.edu/contact">Contact</a>
        <p class="copyright">Computational Genomics Lab © 2015-2018</p>
      </div>
    </div>
    <div class="footer-right">
      <div class="logo-container row">
        <div class="col-sm-4">
          <a target="_blank" href="http://www.ub.edu/web/ub/en/index.html?">
            <img class="ub-logo-footer" src="{% static 'Images/ub-logo.png' %}">
          </a>
        </div>
        <div class="col-sm-4">
          <a target="_blank" href="http://www.ub.edu/ibub/en/index.html">
            <img class="ibub-logo-footer" src="{% static 'Images/ibub-logo.png' %}">
          </a>
        </div>
        <div class="col-sm-4">
          <a target="_blank"  href="http://www.ciberer.es/en">
            <img class="ciberer-logo-footer" src="{% static 'Images/ciberer-logo.png' %}">
          </a>
        </div>
      </div>
    </div>

    <a href="#0" class="cd-top" title="Back to Top">Top</a>
  </div>


<script src="{% static 'js/jquery-min.js' %}"></script>
<script src="{% static 'js/bootstrap-min.js' %}"></script>
<script src="{% static 'js/back-to-top.js' %}"></script>
<script src="{% static 'js/stoopidoverlay.js' %}"></script>
<script src="{% static 'js/overlays.js' %}"></script>

</body>
</html>
